src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -437,14 +433,24 @@ s32 EnMa1_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
 
 void EnMa1_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, void* thisx) {
     EnMa1* this = (EnMa1*)thisx;
-    Vec3f vec = D_80AA16B8;
+    Vec3f vec = { 800.0f, 0.0f, 0.0f };
 
     if (limbIndex == 15) {
```

Comment:
The limb index here and in OverrideLimbDraw should use an enum value from the skeleton. If you to go `object_ma1.xml` and add an `EnumName` attribute to the skeleton, e.g.
`<Skeleton Name="gMalonChildSkel" Type="Flex" LimbType="Standard" EnumName="ChildMalonLimb" Offset="0x8460"/>`
it should generate an enum if you re-run asset extraction (either `make setup` again or with `extract_assets.py -s objects/object_ma1`)
Then replace the raw values that `limbIndex` is being compared to with items from that enum.